### ISSUE TITLE
Bug: Typo error and UI Enhancement for Search Bar Dropdowns

### ISSUE THREAD

**Issue by ashesbloom** [XengShi/materialYouNewTab#359]
### Description  
On the homepage, below the search bar, the left-side element currently displays the text **"Search with"**, which should be corrected to **"Search on"**.  

### Steps to Reproduce  
1. Navigate to [https://xengshi.github.io/materialYouNewTab/](https://xengshi.github.io/materialYouNewTab/).  
2. Observe the text displayed below the search bar on the left-hand side.  

### Screenshots  
![image](https://github.com/user-attachments/assets/1f91075a-cb56-425a-b0b1-e96bb1cd5a1b)  

### Additional Context  
To improve user clarity and functionality, consider separating "Search on" and "Search with" into distinct categories, potentially implemented as dropdown menus:  
- **Search on:** Allow users to select platforms such as YouTube, ChatGPT, LinkedIn, etc.  
- **Search with:** Enable users to choose browser engines like Chrome, Bing, DuckDuckGo, Brave, etc.

**Comment by XengShi** [comment]
`Search on: Allow users to select platforms such as YouTube, ChatGPT, LinkedIn, etc.`
Nice idea 💡

**Comment by TitanX311** [comment]
hello @XengShi @ashesbloom i want to work on the issue. But has it already been resolved?

**Comment by itz-rj-here** [comment]
> hello @XengShi @ashesbloom i want to work on the issue. But has it already been resolved?

@prem-k-r

**Comment by prem-k-r** [comment]
Hi @TitanX311, it's hasn't been resolved. 
Before proceeding can you please share what's your plan? how it will looks like

**Comment by ashesbloom** [comment]
@itz-rj-here yeah go ahead, though I have created a patch but it's buggy and needs polish!

**Comment by TitanX311** [comment]
> Hi @TitanX311, it's hasn't been resolved. Before proceeding can you please share what's your plan? how it will looks like

as mentioned the idea about a dropdown menu with a sliding animation(similar to the "AI Tools" option) and having options like "search with" and "search on". the "search with" option will have google, brave, yahoo etc. and "search on" will have options like spotify, youtube, youtube music etc. Also thought of changing the icon of twitter.
![image](https://github.com/user-attachments/assets/5435da07-354b-4aa3-b9d6-9ebab95eedee)

**Comment by prem-k-r** [comment]
>  Also thought of changing the icon of twitter. 

Well, we have intentionally not updated the icon to 𝕏

**Comment by prem-k-r** [comment]
> as mentioned the idea about a dropdown menu with a sliding animation(similar to the "AI Tools" option) and having options like "search with" and "search on". the "search with" option will have google, brave, yahoo etc. and "search on" will have options like spotify, youtube, youtube music etc. Also thought of changing the icon of twitter. ![image](https://private-user-images.githubusercontent.com/166216400/401523885-5435da07-354b-4aa3-b9d6-9ebab95eedee.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzY0MjQyMTYsIm5iZiI6MTczNjQyMzkxNiwicGF0aCI6Ii8xNjYyMTY0MDAvNDAxNTIzODg1LTU0MzVkYTA3LTM1NGItNGFhMy1iOWQ2LTllYmFiOTVlZWRlZS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwMTA5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDEwOVQxMTU4MzZaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT00NjEyYTdiNzdmY2UxMmVhNWI3Y2ZhODcxYzk2YjM3YjJkN2M0NWY4M2E2NjVlNGMzNTk3YzQ5M2I3MWI2NjVlJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.TjtaA3SdKc7BNDX4wGwqGtJ_2lB-xQrbtiBYza-tNtw)
> 
> Also for your kind info, i am a complete beginner in open source and want to seek guidance. thank you

@XengShi

**Comment by ashesbloom** [comment]
@prem-k-r  I was thinking more like this but with some transition animations or something that goes with the theme.

https://github.com/user-attachments/assets/343c2e3b-a6b7-4f3f-859c-fc7fad224891

**Comment by prem-k-r** [comment]
It's nice, I was also thinking of clickable Search with, instead of dropdown. 

Maybe you can open a PR on draft, people can do the changes there

**Comment by TitanX311** [comment]
> @prem-k-r @TitanX311 I was thinking more like this but with some transition animations or something that goes with the theme.
> 
>  Untitled.video.-.Made.with.Clipchamp.1.mp4 
> PS: if anyone needs this patch ping me up, I'll provide you

please provide the patch
